nrcemt/nanomi_optics/test/test_optimization.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+from nrcemt.nanomi_optics.engine.lens import Lens
+from nrcemt.nanomi_optics.engine.optimization import optimize_focal_length
+
+OPTICAL_DISTANCE = 0.00001
+LAMBDA_ELECTRON = 0.0112e-6
+SCATTERING_ANGLE = LAMBDA_ELECTRON / OPTICAL_DISTANCE
+
+RAYS = [
+    np.array([[0], [SCATTERING_ANGLE]]),
+    np.array([[OPTICAL_DISTANCE], [SCATTERING_ANGLE]])
+]
+
+Y_POINTS = [
+    -4.0968268138824245894e-12,
+    -1.1357412494183741452e-05,
+    -0.0026889413214573167424
+]
+
+Y_POINTS_RED = [
+    57.221005454854299899,
+    0.39533346627242704763,
+    -0.0018439883556115666333
+]
+Y_POINTS_GREEN = [
+    57.218338662064461175,
+    0.39326517213370454362,
+    -0.0040173071880824067631
+]
+CF = [19.67, 6.498, 6]
+LOCATION = [551.6, 706.4, 826.9]
```

Comment:
can you add some explanation for what they are? Are they outputs from Matlab to compare against, are they just configuration for the test cases, etc.